Dockerfile

```diff
@@ -8,37 +8,35 @@ ARG ANDROID_BUILD_VERSION=28
 ARG ANDROID_TOOLS_VERSION=28.0.3
 ARG BUCK_VERSION=2019.05.22.01
 ARG NDK_VERSION=17c
+ARG NODE_VERSION=10.x
 ARG WATCHMAN_VERSION=4.9.0
 
 # set default environment variables
 ENV ADB_INSTALL_TIMEOUT=10
-ENV PATH=${PATH}:/opt/buck/bin/
 ENV ANDROID_HOME=/opt/android
 ENV ANDROID_SDK_HOME=${ANDROID_HOME}
-ENV PATH=${PATH}:${ANDROID_HOME}/emulator:${ANDROID_HOME}/tools:${ANDROID_HOME}/tools/bin:${ANDROID_HOME}/platform-tools
 ENV ANDROID_NDK=/opt/ndk/android-ndk-r$NDK_VERSION
-ENV ANDROID_NDK_HOME=/opt/ndk/android-ndk-r$NDK_VERSION
-ENV PATH=${PATH}:${ANDROID_NDK}
 
-# See https://github.com/debuerreotype/docker-debian-artifacts/issues/24
-RUN mkdir -p /usr/share/man/man1
+ENV PATH=${ANDROID_NDK}:${ANDROID_HOME}/emulator:${ANDROID_HOME}/tools:${ANDROID_HOME}/tools/bin:${ANDROID_HOME}/platform-tools:/opt/buck/bin/:${PATH}
 
-# install system dependencies
-RUN apt-get update -qq && apt-get install -qq -y --no-install-recommends \
+# Install system dependencies
+# See https://github.com/debuerreotype/docker-debian-artifacts/issues/24
+RUN mkdir -p /usr/share/man/man1 \
+    && apt-get update -qq && apt-get install -qq -y --no-install-recommends \
         apt-transport-https \
         curl \
         build-essential \
         file \
         git \
         gnupg2 \
-        openjdk-8-jre \
+        openjdk-8-jdk \
```

Comment:
We may still need jre, last time I remember emulator depend on this.

Reply:
openjdk-8-jdk depends on openjdk-8-jre

Reply:
That's what I think, but from result we still need it. Let's see the CI result.